### PR TITLE
(PDB-4919) Add release notes and env var docs for 6.13.0

### PR DESCRIPTION
**Pull request by rbrw**

### CODE REVIEW

File: documentation/release_notes/release_notes_latest.markdown [hunk 1]
```diff
@@ -21,6 +21,85 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+- PuppetDB now unifies report and resource event clean up during a
+  full garbage collection, instead of handling each in a separate
+  transaction.  This ensures it only waits on the exclusive lock to
+  drop relevant event partitions once.
+  ([PDB-4902](https://tickets.puppetlabs.com/browse/PDB-4902))
+
+- The fact path garbage collection process will now time out after 5
+  minutes by default if it cannot acquire the locks it requires.  See
+  also:
+  https://puppet.com/docs/puppetdb/latest/configure.html#PDB_FACT_PATH_GC_SQL_LOCK_TIMEOUT_MS
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:54]:
```suggestion
  For more information, see [Configuring PuppetDB] (https://puppet.com/docs/puppetdb/latest/configure.html#PDB_FACT_PATH_GC_SQL_LOCK_TIMEOUT_MS)
```

File: documentation/release_notes/release_notes_latest.markdown [hunk 2]
```diff
@@ -21,6 +21,85 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+  ([PDB-4907](https://tickets.puppetlabs.com/browse/PDB-4907))
+
+- SQL commands issued during an attempt to process a command (store a
+  report, update a factset, etc.) will now time out after 10 minutes
+  by default, causing the command to be retried or discarded.  See
+  also:
+  https://puppet.com/docs/puppetdb/latest/configure.html#PDB_COMMAND_SQL_STATEMENT_TIMEOUT_MS
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:61]:
```suggestion
 For more information, see [Configuring PuppetDB](https://puppet.com/docs/puppetdb/latest/configure.html#PDB_COMMAND_SQL_STATEMENT_TIMEOUT_MS)
```

File: documentation/release_notes/release_notes_latest.markdown [hunk 3]
```diff
@@ -21,6 +21,85 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+  ([PDB-4834](https://tickets.puppetlabs.com/browse/PDB-4834))
+
+- PuppetDB will only drop the oldest report or events partition during
+  the normal, periodic gc when there's more than one candidate. This
+  is intended to decrease the length of time we might block pdb
+  operations since with the current arrangement, the drop will attempt
+  to acquire an exclusive lock on the entire table (i.e. reports, not
+  just the partition), and so will block all subsequent access to that
+  table, until it finishes.
+  ([PDB-4901](https://tickets.puppetlabs.com/browse/PDB-4901))
```
Comment (clairecadman) [inline documentation/release_notes/release_notes_latest.markdown:43]: Suggest rewrite of note above: 
PuppetDB only drops the oldest report or events partition during the normal periodic gc when there is more than one candidate. This decreases the length of time PuppetDB blocks operations. Currently, the drop attempts to acquire an exclusive lock on the entire table (i.e. reports, not just the partition) and blocks all subsequent access to that table until it finishes.

**Review by Zak-Kent — APPROVED** [review]
👍

**Pull request merged**